README.md

```diff
@@ -308,8 +308,70 @@ just added.
 ## API
 
 <generated_api_start />
+#### The Jest global environment
+
+In your test files, Jest puts each of these methods and objects into the global environment. You don't have to require or import anything to use them.
+
+  - `afterEach(fn)`
+  - `beforeEach(fn)`
+  - [`check`](https://facebook.github.io/jest/docs/api.html#check)
```

Comment:
For now I will remove them from this top summary but leave the code samples at the bottom. If you really do remove them from Jest, removing the docs entirely is probably the cleanest outcome, but for now this seems reasonable.